### DISCUSSION TITLE
Accepted

### DISCUSSION THREAD

**Discussion by noman2002** [github/india#646] [Students]
Got the acceptance mail today. Anyone got it.

**Comment by lavakush07** [comment]
Yeah,I too got the acceptance mail

**Reply by Ritikace01** [reply]
When did you guys register, what was the date ? I am asking so that I can have an idea about how much time it took for acceptance post registering.

**Reply by lavakush07** [reply]
Yeah it exactly took 1 week for the acceptance

**Reply by Ojasv-Singh-Zeotap** [reply]
i think this acceptance is from github and companies will still shortlist the candidates.

**Reply by lavakush07** [reply]
Yep,Acceptance is only from github side ,we should make the proposal for the projects then companies will evaluate it .After the process of evaluation  companies will shortlist the canditates

**Reply by Deathshadow999** [reply]
I didn't got any email today

**Reply by samyak1999** [reply]
Got the acceptance mail. Regarding the proposal letter what should a candidate write in it ?

**Reply by ArshWalker** [reply]
> Yep,Acceptance is only from github side ,we should make the proposal for the projects then companies will evaluate it .After the process of evaluation companies will shortlist the canditates

Could you please, share the format and points to be mentioned in the project proposal for Reference!

**Reply by atharva137** [reply]
yes please if anyone could share the format for project proposal writing it will be really helpful

**Reply by kyteinsky** [reply]
> https://discord.com/channels/846035377742544897/902850627812597821/917023516849762304
> 
> Follow the mentioned points

Nothing comes up, can you put the text here directly? @samyak1999

**Reply by samyak1999** [reply]
https://gist.github.com/liyasthomas/2ece091639028a7ecaafaade5ae8e2c6

![image](https://user-images.githubusercontent.com/43057462/144954130-eff82a1b-4069-480d-95c1-2bac0ddd069b.png)
